operations/mimir/zone-aware-pod-disruption-budget.libsonnet

```diff
@@ -0,0 +1,24 @@
+{
+  // zpdbTemplate creates new ZoneAwarePodDisruptionBudget resource.
+  // name is the rollout-group
+  zpdbTemplate(name, rolloutGroup, maxUnavailable, ingestStorageEnabled):: {
```

Comment:
[nit] I'd call this something like `newZPDB` or `zpdb`: we're a bit inconsistent with the naming convention here, but we seem to use either of those two patterns.